server/database/db.js

```diff
@@ -0,0 +1,106 @@
+const Sequelize = require('sequelize');
+
+const sequelize = new Sequelize('gallery', 'student', 'student', {
+  // host: 'localhost',
+  dialect: 'mysql',
+  operatorsAliases: false,
+
+  pool: {
+    max: 5,
+    min: 0,
+    acquire: 30000,
+    idle: 10000
+  },
+
+});
+
+sequelize
+  .authenticate()
+  .then(() => {
+    console.log('Connection has been established successfully.');
+  })
+  .catch(err => {
+    console.error('Unable to connect to the database:', err);
+  });
+
+const Interior = sequelize.define('interior', {
+  id: {
+    type: Sequelize.INTEGER,
+    primaryKey: true,
+    autoIncrement: true
+  },
+  fileName: {
```

Comment:
Lines 32-34 could be cleaned up to:
"fileName: Sequelize.STRING",

Nothing wrong with what you have here though, just a cleanup suggestion.